### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by mezis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,17 +19,42 @@ Routemaster comes with, and is automatically integration-tested against
 a Ruby client,
 [routemaster-client](https://github.com/deliveroo/routemaster-client).
 
-## Remote procedure call as an antipattern
+For advanced bus consumers,
+[routemaster-drain](https://github.com/deliveroo/routemaster-drain) can perform
+filtering of event stream
+preemptive caching of resources.
```
Comment (rxbynerd) [inline README.md]: missing `&` btwn these two?

File: README.md [hunk 2]
```diff
@@ -12,17 +19,42 @@ Routemaster comes with, and is automatically integration-tested against
 a Ruby client,
 [routemaster-client](https://github.com/deliveroo/routemaster-client).
 
-## Remote procedure call as an antipattern
+For advanced bus consumers,
+[routemaster-drain](https://github.com/deliveroo/routemaster-drain) can perform
```
Comment (levriero) [inline README.md:23]: 👍

File: README.md (hunk 1 above)
Comment (levriero) [inline README.md]: > can perform filtering of event stream preemptive caching of resources.

I feel like there should be some conjunction here, a comma, and, or something.

File: README.md [hunk 3]
```diff
@@ -12,17 +19,42 @@ Routemaster comes with, and is automatically integration-tested against
...
+preemptive caching of resources.
+
+## The basics
+
+Routemaster lets publisher push events into topics, and subscribers receive
+events about topics they've subscribed to.
+
+Pushing, receiving, and subscribing all happen over HTTP.
+
+![](https://www.dropbox.com/s/qbn1srjjcb8t0vh/Screenshot%202016-08-25%2016.41.54.png?raw=1)
```
Comment (levriero) [inline README.md:33]: :100:

Although I'm not sure if this makes clear that a subscriber will receive batched messages. In this particular example, subscriber 2 may receive N numbers of messages belonging to 3 different topics batched in one request.

File: README.md [hunk 4]
```diff
@@ -12,17 +19,42 @@ Routemaster comes with, and is automatically integration-tested against
...
+
+
+## Rationale
+
+We built Routemaster because existing buses for distributed architecture aren't
```
Comment (levriero) [inline README.md]: s/architecture/architectures

File: README.md [hunk 5]
```diff
@@ -12,17 +19,42 @@ Routemaster comes with, and is automatically integration-tested against
...
+satisfactory to us; either they're too complex to host and maintain, don't
+support key features (persistence), or provide too much rope to hang ourselves
+with.
+
+### Remote procedure call as an antipattern
 
 Routemaster is designed on purpose to _not_ support RPC-style architectures, for
 instance by severely limiting payload contents.
 
+It _only_ supports notifying consumers about lifecycle (CRUD) changes to
```
Comment (levriero) [inline README.md:50]: 👍

**Comment by levriero** [comment]
Some minor comments but other than that can't 👍  enough.

File: README.md (hunk 3 above)
Comment (mezis) [reply README.md:33]: I'll add a clarification.

**Comment by mezis** [comment]
Thanks folks. I'll merge, shout if more clarifications are relevant.

**Pull request merged**